siliconcompiler/schema/schema_cfg.py

```diff
@@ -2649,6 +2604,137 @@ def schema_option(cfg):
             before a run.
             """)
 
+    scparam(cfg, ['option', 'timeout'],
```

Comment:
I like the idea of providing this option for all jobs, not just scheduled ones. Should it have `pernode='optional'`, so we can specify per-task timeouts?